ssms/dataset_generators/lan_mlp.py

```diff
@@ -247,6 +247,23 @@ def _make_kde_data(self, simulations=None, theta=None):
         out[(n_kde + n_unif_up) :, -2] = choice_tmp
         out[(n_kde + n_unif_up) :, -1] = self.generator_config["negative_rt_cutoff"]
         return out.astype(np.float32)
+    
+    def parameter_transform_for_data_gen(self, theta):
+        '''
+        Function to impose constraints on the parameters for data generation.
+        '''
+
+        # For LBA-based models, we need to ensure that the drift rates sum to 1
+        if self.model_config["name"] == "rlwm_lba_race_wo_ndt_v1":
+            theta[0:3] = theta[0:3]/np.sum(theta[0:3])
```

Comment:
Let's just add a few comments here, for clarity? Which parameter subsets doe the softmax call respectively belong to.
 
Might be better to put this parameter transform thing into the model config and apply at the level of `simulator()` call? 

This is essentially 'part of the model' right? You always want to do that when you call the simulator for `rlwm_lba_race_wo_ndt_v1`.

Reply:
Added some comments about the specifics of transforms. 
Yes, generally including abstractions for `simulator()` is a good strategy however, this drift sum constraint is specific to the new LBA class. We worry about this constraint only for data generation here and we do not expect to alter this constraint from a theoretical or experimental perspective. So I think it is better to just hard code for data generation and avoid complicating the model_config. In any case, if the user-supplied drift violates this constraint, the simulator will complain.

Reply:
alright let's keep it for now